nanshe_workflow/data.py

```diff
@@ -94,23 +90,21 @@ def _int_dask_rm_tree(dname):
 def dask_io_remove(name, executor=None):
     name = os.path.abspath(name)
 
-    rm_task = None
-    if not os.path.exists(name):
-        rm_task = dask_rm_nothing()
-    elif os.path.isfile(name):
-        rm_task = dask_rm_file(name)
-        rm_task = dask.delayed(io_remove)(rm_task)
-    elif os.path.isdir(name):
-        rm_task = dask_rm_tree(name)
-        rm_task = dask.delayed(io_remove)(rm_task)
-    else:
-        raise ValueError("Unable to remove path, '%s'." % name)
+    tmp_dir = "tmp_nanshe_workflow_{0}_".format(
+        os.path.splitext(os.path.basename(name))[0]
```

Comment:
Name the temporary directory something nice to give us an idea of what it contains (i.e. some data from the workflow), but still allow random stuff to be added to the end to keep it unique (important for multiple deletions of the same filename back to back).